frameworks/helloworld/src/main/dist/profile-mount-volume.yml

```diff
@@ -0,0 +1,27 @@
+name: {{FRAMEWORK_NAME}}
+scheduler:
+  principal: {{FRAMEWORK_PRINCIPAL}}
+  user: {{FRAMEWORK_USER}}
+pods:
+  hello:
+    count: {{HELLO_COUNT}}
+    tasks:
+      profile:
+        goal: RUNNING
+        cmd: |
+               set -e -x
+               if [ -n "{{{TEST_PROFILE_VOLUME_COMMAND}}}" ]; then
+                   {{{TEST_PROFILE_VOLUME_COMMAND}}} profile-container-path
+               fi
+               echo profile >> profile-container-path/output
+               sleep $SLEEP_DURATION
+        cpus: {{HELLO_CPUS}}
+        memory: {{HELLO_MEM}}
+        env:
+          SLEEP_DURATION: {{SLEEP_DURATION}}
+          VOLUME_PROFILE: {{HELLO_VOLUME_PROFILE}}
+        volume:
+          path: profile-container-path
+          profiles: [{{HELLO_VOLUME_PROFILE}}]
```

Comment:
we have sth similar at https://github.com/mesosphere/dcos-commons/blob/9827ee6fbee1803df47345b3e1b450484142ac7d/frameworks/helloworld/src/main/dist/pod-profile-mount-volume.yml#L14 as well fwiw

Reply:
Yes, this is based on `pod-profile-mount-volume` but uses a single profile volume instead. `pod-profile-mount-volume` also uses a regular persistent volume. This is useful for testing profile volume support in the SDK. `profile-mount-volume` is meant as a minimal configuration for testing profile volumes as part of `hello-world`.